### ISSUE TITLE
Does it work for College Football USA '97?

### ISSUE THREAD

**Issue by jd6-37** [Goldglv/John-Madden-Football-Roster-Editor#1]
Hi, this looks like a great tool. I was wondering if it worked for College Football USA '97 or if I could request that feature.

Thanks!

**Comment by Goldglv** [comment]
Currently doesn't work with Bill Walsh 95 or College Football 97 but I looked at both files, I think I can adjust the editor to work with both.  Just like Madden 98, College 97 has 2 additional numbers in the ROM which I assume are ratings, after their main ratings that don't appear to change anything.  May have something to do with injury or affect the stamina of the player.

Will update when I've got something working with both...

I sent you an email to your yahoo acct with some more details.

**Comment by jd6-37** [comment]
Awesome! Thanks for looking into it. 

Um, I don't think I have access to a yahoo email anymore. Sorry. I only see the issue reply notification email. Do you mind resending it to < redacted >?

**Comment by Goldglv** [comment]
Resent

**Comment by HITTER4252** [comment]
Wondering also if this works for College Football USA 97

**Comment by Faikan3s3** [comment]
@Goldglv Can you make a new editor for the other madden games?

SNES Madden 93, 95, 96, 97 and 98.
Genesis Madden 95, 96 and 97.

**Comment by Goldglv** [comment]
> Wondering also if this works for College Football USA 97

Yes, will be uploading that version of the Editor within the next few days.

**Comment by Goldglv** [comment]
> @Goldglv Can you make a new editor for the other madden games?
> 
> SNES Madden 93, 95, 96, 97 and 98. Genesis Madden 95, 96 and 97.

Yes, this is definitely a possibility but will take some time, is there a specific Console/Version you are looking to play first?  I can start working on that version first...

**Comment by Faikan3s3** [comment]
@Goldglv I'll go with Madden 95 for the SNES. I want to update the names for each player instead of just having numbers.

Thanks BTW. I saw the updated Madden editor. Does it include 95?

**Comment by Goldglv** [comment]
> @Goldglv Can you make a new editor for the other madden games?
> 
> SNES Madden 93, 95, 96, 97 and 98. Genesis Madden 95, 96 and 97.

College Football USA '97 Roster Editor can be found here:

[https://github.com/Goldglv/College-Football-Roster-Editor]

**Comment by Goldglv** [comment]
> > Wondering also if this works for College Football USA 97
> 
> Yes, will be uploading that version of the Editor within the next few days.

College Football USA '97 Roster Editor can be found here:

[College Football USA '97 Roster Editor](https://github.com/Goldglv/College-Football-Roster-Editor)
